torch/_torch_docs.py

```diff
@@ -445,6 +445,32 @@ def merge_dicts(*dicts):
             [ 0.7573, -3.9555, -2.8681]])
 """.format(**common_args, **tf32_notes))
 
+add_docstr(torch.sspaddmm,
+           r"""
+sspaddmm(input, mat1, mat2, *, beta=1, alpha=1, out=None) -> Tensor
+
+This function does exact same thing as :func:`torch.addmm` except the
```

Comment:
I think this can still describe what the function does before referring to addmm.

Follow-up question, should this be sparse.addmm? 

What about something like: 

"Matrix multiplies a sparse tensor mat1 with a dense tensor mat2, then adds the sparse tensor input to the result."?

Then this could add:

"Note: This function is equivalent to torch.addmm, except input and mat1 are sparse."

Reply:
> Follow-up question, should this be sparse.addmm?

No: although the current `torch.sparse.addmm` and `torch.sspaddmm` have the same arguments, one returns a strided tensor and another a sparse tensor.

That said, here's an idea of how to merge these two functions into one, say, to `torch.sparse.addmm`: when the user needs an output with a specific layout, s/he can specify the `out` argument with the desired output. By default, the output would be a strided tensor.

> What about something like:
> 
> "Matrix multiplies a sparse tensor mat1 with a dense tensor mat2, then adds the sparse tensor input to the result."?
> 
> Then this could add:
> 
> "Note: This function is equivalent to torch.addmm, except input and mat1 are sparse."

fixed.

Reply:
Resolving via https://github.com/pytorch/pytorch/issues/45897 .